index.html

```diff
@@ -585,8 +591,65 @@ <h1>Introduction</h1>
     <a href="#json-ld-ctx-usage"></a> and the documentation under the namespace IRIs, e.g.,
     <a href="https://www.w3.org/2019/wot/td">https://www.w3.org/2019/wot/td</a>).
 </p>
+</section>
+<section id="introduction-tm">
+    <h2>Thing Model</h2>
+    
+    <p>One of the main intentions of a <a>Thing Description</a> is to provide a <a>Consumer</a> 
+        with all the details necessary to successfully interact with a <a>Thing</a>.  
+        In some IoT application scenarios, a full detailed <a>Thing Description</a>, e.g., with 
+        communication metadata is not necessary (e.g., IoT ecosystems implicitly handle communication separately),
+        or may not be available because a new entity has not yet been deployed (e.g., IP address is not yet known). 
+        Sometime also a kind of class definition is required that forces capability definitions that shall be 
+        available for all created instances (e.g., large-scale production of new devices).
+    </p> 
+        
+    <p>Addressing such or other scenarios the <a>Thing Model</a> can be used that mainly provides the data model 
+        definitions within <a>Things</a>' <a>Properties</a>, <a>Actions</a>, and/or <a>Events</a> and can potential 
+        be used as template for creating <a>Thing Description</a> instances. In the following a sample 
+        <a>Thing Model</a> is presented that can be seen as a model for the <a>Thing Description</a>
+        instance in <a href="#simple-thing-description-sample"></a>. 
+    </p>
+
+    <pre class="example" title="Thing Model sample" id="td-model-example-lamp">
+        {
+            "@context": ["http://www.w3.org/ns/td"], 
+            "@type" : "tm:ThingModel",
+            "title": "Lamp Thing Model",
+            "properties": {
+                "status": {
+                    "description": "current status of the lamp (on|off)",
+                    "type": "string",
+                    "readOnly": true
+                }
+            },
+            "actions": {
+                "toggle": {
+                    "description": "Turn the lamp on or off"
+                }
+            },
+            "events": {
+                "overheating": {
+                    "description": "Lamp reaches a critical temperature (overheating)",
+                    "data": {"type": "string"}
+                }
+            }
+        }
+        </pre>
+        <p>
+            <a>Thing Model</a> definitions are identified by the <code>"@type": "tm:ThingModel"</code>. 
+            As the example shows, it does not provide details about a single <a>Thing</a> instance due to the 
+            lack of communication and security metadata provided.        
+        </p>
+
+        <p>
+            This specification presents a procedure for deriving valid <a>Thing Description</a> instances 
+            and the feature tools that can be used to design a <a>Thing Model</a> that also allows, among 
```

Comment:
it is not super clear. Do you mean that this specification features tools to do X, like when one says "Die Hard 4, featuring Bruce Willis"?

I think it should be something like "other design concepts that allow to, among others, overwriting, extending, and reusing existing <a>Thing Model</a> definitions."

I think among other should be removed since it makes the sentence sound complicated.

Reply:
done